po-man/install-translations.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+function usage()
+{
+    cat << HEREDOC
+
+ Usage:
+  $PROGRAM --mandir <directory> --mansrcdir <directory> [<man page name>...]
+
+Install translated man pages.
+
+ Options:
+  --help                             show this help message and exit
+  --mandir <mandir>                  directory in which to install the man pages, usually share/man
+  --mansrcdir <mansrcdir>            directory containing the man pages to install
+
+ Environment Variables:
+  MESON_INSTALL_DESTDIR_PREFIX       install destination directory
```

Comment:
`DESTDIR` is more generic and commonly used :-)

Reply:
Agreed. `MESON_INSTALL_DESTDIR_PREFIX` is specially handled by Meson to join absolute paths as documented [here](https://mesonbuild.com/Release-notes-for-0-37-0.html#improvements-to-install-scripts), but I think that's mostly for Windows which isn't as much of a concern for `util-linux`. I'll change it.

Reply:
Done.